README.md

```diff
@@ -1,8 +1,10 @@
 # Engagement Support Automation Backend
 Develop Branch :
```

Comment:
Nice job! This phrase: `Develop Branch`, is it necessary to be here? What if we remove it

Reply:
it's not necessary. I think we can remove it